### ISSUE TITLE
Linker error when trying to compile

### ISSUE THREAD

**Issue by rapperskull** [emukidid/swiss-gc#453]
I'm trying to compile using the latest version of devkitPro, but I get the following error:

> $ make
> Building on Windows_NT
> make[1]: Entering directory '/home/Rapper_skull/swiss-gc/cube/swiss'
> clean ...
> make[1]: Leaving directory '/home/Rapper_skull/swiss-gc/cube/swiss'
> make[1]: Entering directory '/home/Rapper_skull/swiss-gc/pc/usbgecko'
> make[1]: Leaving directory '/home/Rapper_skull/swiss-gc/pc/usbgecko'
> make[1]: Entering directory '/home/Rapper_skull/swiss-gc/cube/patches'
> Building Stub ...
> collect2.exe: error: ld returned 53 exit status
> make[1]: *** [Makefile:40: stub.bin] Error 1
> make[1]: Leaving directory '/home/Rapper_skull/swiss-gc/cube/patches'
> make: *** [Makefile:48: compile-patches] Error 2

I've compiled and installed your fork of libogc doing a `make install`.

**Comment by Extrems** [comment]
This may be caused by building from `/home` in MSYS2.

**Comment by rapperskull** [comment]
Where should I build from? I tried cloning to /opt but without changes.

**Comment by rapperskull** [comment]
This is actually still an issue. I don't know why you closed it.

**Comment by Extrems** [comment]
Is it? How?

**Comment by rapperskull** [comment]
I'm still getting the same error under mSys. I'm using WSL to compile under Windows, but it has the unfortunate side effect of not supporting 32-bit executables, so I can't `make all`, since _dollz3_ is a 32-bit program.

**Comment by Extrems** [comment]
Are you using standalone MSYS2 or bundled with devkitPro?

**Comment by rapperskull** [comment]
I used _devkitProUpdater-3.0.3_ to set-up everything, so bundled with devkitPro.

**Comment by Extrems** [comment]
Yeah, but this one component is using link-time optimization. I'm using standalone MSYS2.

**Comment by rapperskull** [comment]
I set up a standalone msys, but I'm having the same issue.

**Comment by rapperskull** [comment]
Solved after seeing this: https://github.com/devkitPro/pacman-packages/issues/199
The problem is completely different, but adding _$DEVKITPRO/tools/bin_ in front of the PATH solves the issue.
I am now however struggling with _cygwin1.dll_, needed for _mkisofs.exe_. I tried an older version that shipped with the program itself, but I get the following error: `conflicting versions of cygwin1.dll detected.  Use only the most recent version.`, while mkisofs simply doesn't work with newer versions.
